lib/array_destructer.rb

```diff
@@ -0,0 +1,15 @@
+module MethodExecution
+  module ArrayDestructer
```

Comment:
Don't use `-er` naming.   Whis is this method not under `module RuboClaus` namespace?  What about `RubyClaus::Destructure`